python/hsfs/core/statistics_engine.py

```diff
@@ -78,7 +78,7 @@ def profile_statistics(metadata_instance, feature_dataframe):
                 "to the online storage of a feature group.",
                 category=util.StatisticsWarning,
             )
-            return None
+            return "{}"
```

Comment:
Is this change related to the PR`?

Reply:
I had an error when testing this PR. If the dataframe is empty, it causes error in the backend.

Reply:
Was this an interegration test that failed?